features/karaf/pom.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.eclipse.smarthome</groupId>
+    <artifactId>features</artifactId>
+    <version>0.8.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>karaf</artifactId>
+  <packaging>feature</packaging>
+
+  <properties>
+  	<karaf.version>4.0.3</karaf.version>
```

Comment:
The bundle org.eclipse.smarthome.io.console.karaf is using "4.0.0.M2".
Shouldn't we define one global karaf version in the root pom (so moving this property to the root one) and use the one property in both artifacts?